velox/exec/AssignUniqueId.cpp

```diff
@@ -44,6 +44,9 @@ AssignUniqueId::AssignUniqueId(
 
   resultProjections_.emplace_back(0, numColumns - 1);
   results_.resize(1);
+
+  // Ensure the rowIdCounter_ is initialized
```

Comment:
Wouldn't it be sufficient to just initialize rowIdCounter_ and maxRowIdCounterValue_ to zero?

Reply:
@mbasmanova U r right, updated